src/SFA.DAS.ApplyService.Data/ApplyRepository.cs

```diff
@@ -127,7 +127,28 @@ public async Task SubmitApplication(Guid applicationId, ApplyData applyData, Gui
             }
         }
 
+        public async Task<bool> ChangeProviderRoute(Guid applicationId, int ProviderRoute)
+        {
+            var application = await GetApplication(applicationId);
+            var applyData = application?.ApplyData;
+
+            if (application != null && applyData?.ApplyDetails != null)
+            {
+                applyData.ApplyDetails.ProviderRoute = ProviderRoute;
 
+                using (var connection = new SqlConnection(_config.SqlConnectionString))
+                {
+                    await connection.ExecuteAsync(@"UPDATE Apply
```

Comment:
Can this be simplied to use JSON_MODIFY?

Reply:
yeah it can... done 👍